_includes/details-code.html

```diff
@@ -30,7 +30,7 @@
         aria-expanded="true" aria-controls="{{ include.title | slugify | append: '-details' }}">
         Documentation
       </button>
-      <div id="{{ include.title | slugify  | append: '-details' }}" class="nz-accordion-content styleguide-border">
+      <div id="{{ include.title | slugify  | append: '-details' }}" class="nz-accordion-content styleguide-border" role="listitem" aria-label="accordion">
```

Comment:
is accordian what the screen reader says? It might not give the user enough info on what this widget is.